documentation/README.md

```diff
@@ -0,0 +1,80 @@
+# How we write docs
+
+This guide explains the framework we use for writing documentation for all our projects.
+
+## Classification
+
+We can classify the resource types into four groups:
+
+- Getting started: Provide a quick route to begin using the project; their objective is to teach the basics to get started.
+- User Guide: Provide in-depth walkthroughs to use the project; they are more advanced versions than the "Getting started" guides, but their primary purpose is also teaching.
+- Cookbook: Recipes to achieve a specific goal. They provide short and to-the-point examples to perform common tasks. Their objective is to be a shortcut so the user doesn't have to gather all the pieces from reading the API reference.
+- API reference: They provide detailed documentation about the project's API. They are automatically generated from docstrings and document function arguments. They also contain basic examples that users can copy paste.
+
+The following sections describe each group in more detail.
+
+### Getting started
+
+We write these tutorials as Jupyter notebooks; this makes authoring easier and allows users to download and try them with Binder. The language should be pretty straightforward, and we should omit non-essential details; the objective here is to teach the absolute basics.
+
+Depending on the project, sometimes we might decide to have a single notebook be the "Getting started" tutorial; other times, we might have multiple ones, for example, one for each module.
+
+### Guidelines
```

Comment:
This doesn't appear in the main 4 above.

Reply:
it's not one of those four, the heading level was wrong. fixed